scripts/src/main/resources/scripts/command/java

```diff
@@ -91,8 +87,214 @@ function doCreate() {
       cd "${DEVON_IDE_HOME}/workspaces/main" || exit 255
     fi
   fi
-  doMavenArchetype "${@}"
-  exit ${?}
+
+local framework
+local mavenArgs
+local package
+local artifactId
+while [[ -n "${1}" ]]
+do
+  if [[ "${1}" == "-f" ]] || [[ "${1}" == "-framework" ]]  
+  then 
+    if [[ "${2,,}" == "quarkus" ]]
+    then 
+      framework="quarkus"
+      shift
+    elif [[ "${2,,}" == "spring" ]]
+    then 
+      framework="spring"
+      shift
+    fi
+  elif [[ "${1:0:8}" == "-Ddbtype" ]]
+  then 
+    dbtype=${1:9}
+  elif [[ "${1:0:2}" == "-D" ]]
+  then
+      mavenargs="${mavenargs} ${1}"
+  elif [[ "${1}" =~ [a-z]+[.][a-z]+([.][a-z]+)* ]]
+  then 
+      package="${package} ${1}"
+  elif [[ "${1}" =~ [^.]* ]]
+  then
+    ## some better definition for a artifactID TODO
+      artifactId+=${1}
```

Comment:
To explain my comment. This is the documentation of the current behavior:
https://github.com/devonfw/ide/blob/master/documentation/java.asciidoc#create

With the change the arguments given after  `devon java create` are now detected by some semantic magic.
 `[[ "${1}" =~ [a-z]+[.][a-z]+([.][a-z]+)* ]]` will assume the package/groupdId is specified.

With the old behavior you can also do `devon java create com.example.domain.myapp com.example.group demo.app`
what will result in:
* package/groupId=com.example.domain.myapp com.example.group`
* artifactId=demo.app

This will not work anymore as before after this change.

Reply:
We can still extend the CLI and add more expressive sematics like `-DartifactId=demo.app -DgroupId=com.example.group -Dpackage=com.example.domain.myapp` or `-a demo.app -g com.example.group -p com.example.domain.myapp`.

However, if there is no good reason for breaking the existing CLI behavior we should avoid doing this.